### PR TITLE
add Egeria blog post to blogs

### PR DESCRIPTION
**Pull request by merobi-hub**
This change adds Mandy's blog post to the blog.

Signed-off-by: Michael Robinson <merobi@gmail.com>

### CODE REVIEW

**Comment by merobi-hub** [comment]
I believe it's ready for review, @rossturk

File: contents/blog/openlineage-egeria/index.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+date: 2022-04-15
+author: Mandy Chessel
+image: ./image.svg
+banner: ./banner.svg
+description: The Egeria project uses OpenLineage to enhance its production of holistic metadata about an organization's operations.
+---
+## OpenLineage Support in Egeria 
+
+[Egeria](https://egeria-project.org) is a sister open source project to OpenLineage in the LF AI and Data Foundation. Egeria provides Open Metadata and Governance standard types and integration technology to exchange metadata between different technologies. It stitches together different standards to create a complete landscape of metadata about an organization’s digital operations.
+OpenLineage is very welcome to the Egeria team since it defines a standard for dynamic lineage capture.  This means Egeria can capture open lineage events to detect new assets and activity around them, link this new knowledge into the existing metadata and distribute it to the open metadata ecosystem.
```
Comment (rossturk) [inline contents/blog/openlineage-egeria/index.mdx:12]: The Gatsby markdown parser needs a blank line in between paragraphs here (and the following three) or they won't be separated by any space in the final page.

**Review by rossturk — CHANGES_REQUESTED** [review]
Minor spacing update. Also, I think we should make some new artwork so we don't have to reuse - I can show you how I've been doing it.

File: contents/blog/openlineage-egeria/index.mdx (hunk 1 above)
Comment (merobi-hub) [reply contents/blog/openlineage-egeria/index.mdx:13]: Thanks -- made the change.

**Review by rossturk — APPROVED** [review]

**Comment by rossturk** [comment]
Hey @mandy-chessell! Here's how it turned out - screenshot attached. Let us know if it looks good to go.

Screenshot 2022-04-19 at 19-02-27 OpenLineage Support in Egeria

**Review by mandy-chessell — APPROVED** [review]

**Pull request merged**